src/test/java/org/gridsuite/study/server/EquipmentInfosServiceTests.java

```diff
@@ -186,6 +186,34 @@ public void testDeleteAllEquipmentInfos() {
         assertEquals(0, equipmentInfosService.getEquipmentInfosCount());
     }
 
+    @Test
+    public void testGetOrphanEquipmentInfosNetworkUuids() {
+        // index some equipment infos as orphan
+        UUID orphanNetworkUuid = UUID.randomUUID();
+        EquipmentInfos orphanLoadInfos = EquipmentInfos.builder().networkUuid(orphanNetworkUuid).id("id").name("name").type("LOAD").voltageLevels(Set.of(VoltageLevelInfos.builder().id("vl").name("vl").build())).build();
+        UUID orphanNetworkUuid2 = UUID.randomUUID();
+        EquipmentInfos orphanVlInfos = EquipmentInfos.builder().networkUuid(orphanNetworkUuid2).id("id").name("name").type("VOLTAGE_LEVEL").voltageLevels(Set.of(VoltageLevelInfos.builder().id("vl").name("vl").build())).build();
+
+        equipmentInfosService.addEquipmentInfos(orphanLoadInfos);
+        equipmentInfosService.addEquipmentInfos(orphanVlInfos);
+
+        // get the orphan network uuids
+        List<UUID> orphanNetworkUuids = equipmentInfosService.getOrphanEquipmentInfosNetworkUuids(List.of(NETWORK_UUID));
+
+        // check that the orphan network uuids are returned
+        assertEquals(2, orphanNetworkUuids.size());
+        assertTrue(orphanNetworkUuids.contains(orphanNetworkUuid));
+        assertTrue(orphanNetworkUuids.contains(orphanNetworkUuid2));
+
+        // delete the orphan equipment infos
+        equipmentInfosService.deleteAllByNetworkUuid(orphanNetworkUuid);
+        equipmentInfosService.deleteAllByNetworkUuid(orphanNetworkUuid2);
+
+        // check that the orphan network uuids are not returned anymore
+        orphanNetworkUuids = equipmentInfosService.getOrphanEquipmentInfosNetworkUuids(List.of(NETWORK_UUID));
+        assertEquals(0, orphanNetworkUuids.size());
+    }
```

Comment:
It could be nice to add a non orphan element in ES to check it's not returned here